src/config/clientRegistry.en.json

```
@@ -273,8 +279,14 @@
       "description": "Make a booking to visit someone in prison.",
       "link_text": "Go to your visit someone in prison account",
       "link_href": "https://prison-visits.service.justice.gov.uk/"
+    },
+    "S1hl5G31dSsMYqPaOuiRVOLhBX0": {
+      "header": "Update your Active Travel England capital schemes",
+      "description": "Review and update your travel capital schemes.",
+      "link_text": "Go to your travel capital schemes account",
+      "link_href": "https://update-your-capital-schemes.activetravelengland.gov.uk/"
     }
-  },
+},
```

The PR looks good to me other than this bit of indentation which was removed


```suggestion
  },
```

---

Thanks Dana. We need to add a pre-commit to the backend as well to lint and prettify all changes so this doesn't happen again.